cw_monero/lib/monero_wallet_addresses.dart

```diff
@@ -109,7 +136,10 @@ abstract class MoneroWalletAddressesBase extends WalletAddresses with Store {
         accountIndex: accountIndex,
         defaultLabel: defaultLabel,
         usedAddresses: usedAddresses.toList());
-    subaddress = (subaddressList.subaddresses.isEmpty) ? Subaddress(id: 0, address: address, label: defaultLabel) : subaddressList.subaddresses.last;
+    subaddress = (subaddressList.subaddresses.isEmpty) ? Subaddress(id: 0, address: address, label: defaultLabel, balance: '0', txCount: 0) : subaddressList.subaddresses.last;
```

Comment:
`allAddresses` and `getAllUnusedAddresses` are reversed, let's check if the list is also affected or not (i.e should it be `.first` or keep it as is `.last`)

Reply:
It is a normal ObservableList<> so shouldn't be reversed - keeping it last makes sense.

Reply:
https://github.com/cake-tech/cake_wallet/blob/b6089a2dd8d57533f5d7252333c11bb49c9dd466/cw_monero/lib/monero_subaddress_list.dart#L64-L65

Reply:
ok thanks for looking into it